lib/cdo/slack.rb

```diff
@@ -147,6 +148,18 @@ def self.message(text, params={})
     end
   end
 
+  def self.join_room(name)
+    response = open(
+      'https://slack.com/api/channels.join'\
+      "?token=#{SLACK_TOKEN}"\
+      "&name=#{name}"\
```

Comment:
Looks like an extra trailing slash here.

Reply:
I was following a pattern I saw elsewhere in the file (i.e. line 51), though I see we're not consistent (line 84 does not have one).